pom.xml

```diff
@@ -544,6 +554,11 @@
             <version>1.86</version>
         </dependency>
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
Not sure if we need to add new dependency. looking at dependency:tree, 
```
[INFO] +- io.jenkins.blueocean:blueocean-web:jar:1.3.0-beta-1-SNAPSHOT:compile   
[INFO] |  +- org.jenkins-ci.plugins:metrics:jar:3.1.2.10:compile                 
[INFO] |  |  +- io.dropwizard.metrics:metrics-core:jar:3.1.2:compile             
[INFO] |  |  +- io.dropwizard.metrics:metrics-servlet:jar:3.1.2:compile          
[INFO] |  |  +- io.dropwizard.metrics:metrics-healthchecks:jar:3.1.2:compile     
[INFO] |  |  +- io.dropwizard.metrics:metrics-jvm:jar:3.1.2:compile              
[INFO] |  |  +- io.dropwizard.metrics:metrics-json:jar:3.1.2:compile             
[INFO] |  |  \- org.jenkins-ci.plugins:jackson2-api:jar:2.5.4:compile   
```

metrics plugin gets loaded before GitHub and thats causing older version to get resolved first by `maven hpi:run`. and then it fails to load GitHub api plugin as it needs newer version.

```
java.io.IOException: GitHub API Plugin v1.86 failed to load.
 - Jackson 2 API Plugin v2.5.4 is older than required. To fix, install v2.7.3 or later.
	at hudson.PluginWrapper.resolvePluginDependencies(PluginWrapper.java:621)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:517)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
```
Why not exclude jacson2-api version from metrics dependency?